daemon/indexing/apollo/index.js

```diff
@@ -4,37 +4,268 @@ var search = require('../lib/search.js')
 var db = require('../lib/db.js')
 
 /*
- * Implentation of a GraphQL server using the Apollo framework.
- * See https://www.apollographql.com/server
+ * Implementation of the Origin GraphQL server.
+  * Uses the Apollo framework: https://www.apollographql.com/server
  *
  */
 
 // Type definitions define the "shape" of the data and specify
 // which ways the data can be fetched from the GraphQL server.
 const typeDefs = gql`
+  ######################
+  #
+  # Query output schema.
+  #
+  ######################
+
+  # When querying a set of items, the output is a page.
+  interface OutputPage {
```

Comment:
I now the pagination is not planned to be implemented yet, but since we have the interface laid out I would alter it a bit. When displaying paginated elements the DAPP will need to be aware of the current page & the maximum number of pages. For that reason I would change the interface to: 
```
interface OutputPage {
  pageNumber: Int # Current page number
  lastPageNumber: Int # The number of the last page
  }
```

Reply:
Sounds good. Changed to the following to be consistent with the Page type used as input:
    num: Int!      # Current page number
    size: Int!     # Size of pages.
    total: Int!    # Total number of pages available.